staging/src/k8s.io/apiserver/pkg/server/options/audit.go

```diff
@@ -32,10 +32,28 @@ import (
 	"k8s.io/apiserver/pkg/features"
 	"k8s.io/apiserver/pkg/server"
 	utilfeature "k8s.io/apiserver/pkg/util/feature"
+	pluginbuffered "k8s.io/apiserver/plugin/pkg/audit/buffered"
 	pluginlog "k8s.io/apiserver/plugin/pkg/audit/log"
 	pluginwebhook "k8s.io/apiserver/plugin/pkg/audit/webhook"
 )
 
+const (
+	// ModeBatch indicates that the webhook should buffer audit events
+	// internally, sending batch updates either once a certain number of
+	// events have been received or a certain amount of time has passed.
+	modeBatch = "batch"
+	// modeBlocking causes the webhook to block on every attempt to process
+	// a set of events. This causes requests to the API server to wait for a
+	// round trip to the external audit service before sending a response.
+	modeBlocking = "blocking"
+)
+
+// AllowedModes is the modes known by this webhook.
+var allowedModes = []string{
```

Comment:
I still like bool flag better, but if you want to have a list of modes, please move to the buffered backend module

Reply:
move blocking mode there looks not appropriate

Reply:
buffered backend is only one  of the two modes

Reply:
OK, I see your point, if you want to leave the list of modes, it's possible to keep them here

However now I actually think that it doesn't even make sense to have this list: what other modes do you imagine? IMO it should be controlled by a bool flag in the options

Reply:
No other mode now. Generally speaking, sync and async mode are enough.